serveradmin/dataset/commit.py

```diff
@@ -24,45 +24,60 @@ def __init__(self, commit, user):
         self.user = user
 
         if 'deleted' in commit:
-            self.deleted_servers = commit['deleted']
+            self.deleted = commit['deleted']
```

Comment:
I am renaming this particular variable, though we would be able to rename the "deleted" on the client-server protocol.